test/cp.js

```diff
@@ -757,7 +757,7 @@ test('should not overwrite recently created files (not give error no-force mode)
   t.is(shell.cat(`${t.context.tmp}/file1`).toString(), 'test1');
 });
 
-test('should copy fifos contents', t => {
+test('should copy fifo contents', t => {
```

Comment:
Yep, sounds right :-) (Assuming the srcFile symlink has already been resolved if `options.followsymlink === true`.) Perhaps it's worth passing the whole `options` object though? (Thinking in terms of flexibility, eg if more flags are added (eg `-p`))

(And I agree with your perspective on this: the whole reason I opened this PR in the first place was because it tripped me up, and staying internally consistent would have saved me whereas "acting like cp" wouldn't. Just hadn't quite come to that realisation yet :-p )